scripts/jenkins/fpga/logs.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+#
+# Copyright 2019 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Show all logs related with fpga.
+
+set -o pipefail
+set -o xtrace
+set -o errexit
+
+kubectl logs $(kubectl get pods | grep -i test-fpga-region | awk '{print $1}')
```

Comment:
nit: you don't need to use -i  with grep here.

Reply:
Fixed